appshell/cefclient_mac.mm

```diff
@@ -650,7 +662,6 @@ - (void)createApp:(id)object {
   content_rect = [mainWnd contentRectForFrameRect:[mainWnd frame]];
 
   // Configure the rest of the window
-  [mainWnd setTitle:WINDOW_TITLE];
```

Comment:
Drawing of window title is going to be taken care of `CustomToolBarView`. Actually Apple officially dropped support for installing views in `NSThemeFrame` and `CustomToolBarView` is getting added as a subview inside `NSThemFrame`. I think going forward we need to investigate and figure out how to  officially get a dark toolbar. There is a `[mainWnd setAppearance: NSAppearanceNameVibrantLight]` but that is going to give UI toolbar a more darker look. But this is the actual way to do it. Also there is a way to generate custom appearances and install it via `setAppearance`. That is something I will investigate in the future. 

For now to answer your question, OS is drawing the title along with `CustomTitleBarView` and hence need to go suppress one of them.